Kickstarter-iOS/Features/RewardsCollection/Controller/WithShippingRewardsCollectionViewController.swift

```diff
@@ -180,25 +180,15 @@ final class WithShippingRewardsCollectionViewController: UICollectionViewControl
     self.viewModel.outputs.goToAddOnSelection
       .observeForControllerAction()
       .observeValues { [weak self] data in
-        if featureNoShippingAtCheckout() {
-          self?.goToNoShippingAddOnSelection(data: data)
-        } else {
-          self?.goToAddOnSelection(data: data)
-        }
+        self?.goToNoShippingAddOnSelection(data: data)
       }
 
-    self.viewModel.outputs.goToPledge
+    self.viewModel.outputs.goToCustomizeYourReward
```

Comment:
Updated the name of this output because this is called anytime a nonshippable reward is selected taking the user to the screens that only have the pledge amount or bonus amount selectors shown.